apps/keyboard/test/unit/imes/latin/worker_test.js

```diff
@@ -569,11 +552,11 @@ suite('Latin worker', function() {
     });
 
     test('Non existing word should not be matched', function(next) {
-      prediction('sadjasuufehwuefhwejfd', [, null, null, null], next);
+      prediction('sadjasuufehwuefhwejfd', [null, null, null], next);
     });
 
     test('$ should not yield autosuggest', function(next) {
-      prediction('$', [, null, null, null], next);
+      prediction('$', [null, null, null], next);
```

Comment:
Actually, I don't understand why we could not define it as simply `[]`.
@mnjul, could you help shed some light here?
 Thanks.